<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemDialog</class>
 <widget class="QDialog" name="AddItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="productDataLabel">
       <property name="font">
        <font>
         <family>Pyidaungsu</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ကုန်ပစ္စည်းအချက်အလက်</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Pyidaungsu</family>
        </font>
       </property>
       <property name="text">
        <string>ပစ္စည်းအမှတ်</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="productID"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Pyidaungsu</family>
        </font>
       </property>
       <property name="text">
        <string>ပစ္စည်းနာမည်</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="productName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelPicture">
       <property name="font">
        <font>
         <family>Pyidaungsu</family>
        </font>
       </property>
       <property name="text">
        <string>ဓါတ်ပုံ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblImagePreview">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectImage">
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="labelPicture_2">
       <property name="font">
        <font>
         <family>Pyidaungsu</family>
        </font>
       </property>
       <property name="text">
        <string>လက်ရှိ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="productAmount"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Pyidaungsu</family>
        </font>
       </property>
       <property name="text">
        <string>ဈေးနှုန်း</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="productPrice"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="addBtn">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addBtn</sender>
   <signal>accepted()</signal>
   <receiver>AddItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addBtn</sender>
   <signal>rejected()</signal>
   <receiver>AddItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
